features/step_definitions/smokey_steps.rb

```diff
@@ -39,6 +39,10 @@
   @authenticated_as_client = true
 end
 
+Given /^I have the smokey run identifier$/ do
+  $smokey_run_id ||= SecureRandom.hex
```

Comment:
Interesting solution! I guess if smokey crashes after publishing a doc but before deleting it, we'll have residual docs in our integration / staging environments. This is probably fine since they get wiped daily, no need to go further 🤔